src/WebHelper.js

```diff
@@ -70,7 +70,7 @@ class WebHelper extends Helper {
 
                     nets({url: url}, (err, resp, body) => {
                         // body is a Buffer
-                        if (err) {
+                        if (err || String(resp.statusCode).substr(0, 1) !== '2') {
```

Comment:
I like the idea but I would prefer not to convert the integer into a `String`. How about either:
- check if `resp.statusCode` is between 200 and 299, inclusive, or
- check if the integer part of `resp.statusCode / 100` is exactly 2